### PR TITLE
feat: implement GT opcode

### LINKED ISSUE #16: Opcode: GT

### PR DESCRIPTION
**Pull request by Danikim01**
closes #16 
This PR implements the code generation for the GT opcode

### CODE REVIEW

**Comment by MegaRedHand** [comment]
Please merge with main

File: src/opcodes.rs
Comment (Oppen) [inline src/opcodes.rs:1]: This file was renamed to `program.rs`.

Comment (Danikim01) [reply src/opcodes.rs:1]: > This file was deleted and the `enum` moved to `program.rs`.

i merged the changes from main

Comment (Oppen) [reply src/opcodes.rs:1]: I think you possibly kept the file during merge conflict resolutions by mistake. In main there is no [src/opcodes.rs](https://github.com/lambdaclass/evm_mlir/blob/main/src/opcodes.rs), and the `enum` exists only in [src/program.rs](https://github.com/lambdaclass/evm_mlir/blob/main/src/program.rs).

Comment (Danikim01) [reply src/opcodes.rs:1]: > I think you possibly kept the file during merge conflict resolutions by mistake. In main there is no [src/opcodes.rs](https://github.com/lambdaclass/evm_mlir/blob/main/src/opcodes.rs), and the `enum` exists only in [src/program.rs](https://github.com/lambdaclass/evm_mlir/blob/main/src/program.rs).

resolved

File: README.md [hunk 1]
```diff
@@ -75,7 +76,7 @@ An EVM-bytecode to machine-bytecode compiler using MLIR and LLVM.
 1. (0x09) MULMOD
 1. (0x0A) EXP
 1. (0x0B) SIGNEXTEND
-1. (0x11) GT
+1. (0x10) LT
```
Comment (Oppen) [inline README.md:79]: This fell through the cracks when fixing conflicts it seems.
```suggestion
```

Comment (Danikim01) [reply README.md]: > This fell through the cracks when fixing conflicts it seems.

resolved

**Review by Oppen — APPROVED** [review]

**Pull request merged**